src/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -4150,6 +4172,66 @@ public static void parseTownTrustCommand(CommandSender sender, String[] args, @N
 		town.save();
 	}
 
+	public static void parseTownTrustTownCommand(CommandSender sender, String[] args, @Nullable Town town) throws TownyException {
+
+		if (args.length < 1
+			|| args.length < 2 && (args[0].equalsIgnoreCase("add") || args[0].equalsIgnoreCase("remove"))
+			|| args.length == 1 && !args[0].equalsIgnoreCase("list")) {
+			HelpMenu.TOWN_TRUSTTOWN_HELP.send(sender);
+			return;
+		}
+		
+		if (town == null && sender instanceof Player player) {
+			town = getTownFromPlayerOrThrow(player);
+		}
+		if (args[0].equalsIgnoreCase("list")) {
+			TownyMessaging.sendMessage(sender, TownyFormatter.getFormattedTownyObjects(Translatable.of("status_trustedlist").forLocale(sender), new ArrayList<>(town.getTrustedTowns())));
+			return;
+		}
+
+		checkPermOrThrow(sender, PermissionNodes.TOWNY_COMMAND_TOWN_TRUSTTOWN.getNode());
+
+		Town trustTown = getTownOrThrow(args[1]);
+
+		if (args[0].equalsIgnoreCase("add")) {

```

Comment:
```suggestion
		if ("add".equalsIgnoreCase(args[0])) {
```
Literal "add" is argument of 'equalsIgnoreCase()', instead of its qualifier

Reply:
Have implemented change as suggested in commit.